ballerina/types.bal

```diff
@@ -259,3 +259,560 @@ public enum LogLevel {
     @display {label: "OFF"}
     OFF
 }
+############################################################
+# SQL Rule
+#
+# + filter - Represents a filter which is a composition of an expression and an action that is executed in the pub/sub pipeline.
+# + action - Represents set of actions written in Sql language-based syntax
+public type SqlRule record {
```

Comment:
Need to add labels for all the newly added records

Reply:
Addressed in [f0bbe24](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/f0bbe24dab6c69c66ba7ae68ca343a18c0eaf18b).